core-java-modules/core-java-15/pom.xml

```diff
@@ -51,8 +51,8 @@
                 <configuration>
                     <release>${maven.compiler.release}</release>
                     <compilerArgs>--enable-preview</compilerArgs>
-                    <source>15</source>
-                    <target>15</target>
+                    <source>14</source>
+                    <target>14</target>
```

Comment:
Is this correct? Why doesn't it use 15?

Reply:
Good question. Assuming a copy/paste error from another module? Regardless I updated to 15.